include/soloud.h

```diff
@@ -37,7 +37,7 @@ freely, subject to the following restrictions:
 #endif
 
 #if !defined(DISABLE_SSE)
-#ifdef _MSC_VER
+#if defined(__x86_64__) || defined( _M_X64 ) || defined( __i386 ) || defined( _M_IX86 )
```

Comment:
Not sure about this.. __i386? Also, does msvc set one (or more) of those? Generally speaking I suppose just detecting intel architecture is fine as everything out there supports at least SSE2 nowadays, but to be pedantic, __i386 covers a lot of old stuff that doesn't.. =)